src/cromshell/utilities/workflow_id_utils.py

```diff
@@ -0,0 +1,105 @@
+import csv
+import logging
+
+from cromshell.utilities import cromshellconfig, io_utils
+
+LOGGER = logging.getLogger(__name__)
+
+
+def resolve_workflow_id(cromshell_input: str, submission_file_path: str) -> str:
+    """
```

Comment:
It's not inforced but expected that the submission file provided for this [function comes](https://github.com/broadinstitute/cromshell/pull/186/files#diff-b6d9261c391f02014831d7e67689253bd94dc89c20fca63e9fc4e22141de244bR29) from the configuration.py file. That file has a function that will automatically search for the submission file in the cromshell hidden directly if one doesn't exist it will create a new one. In this situation where the path is coming from another portion of the code is it necessary to check if the file exists? (it shouldn't happen but I can see a situation where this function is provided the submission file directly)